### ISSUE TITLE
Review Home & Landing Page User Research

### ISSUE THREAD

**Issue by meglake** [NCIOCPL/ncids#480]
*Description of what is to be accomplished.*
Review Home & Landing page user research: https://lion.box.com/s/gu7ujehwy2qnpfh9jokmxk1zqcnvg9pj and determine if any of the findings warrant any changes to the current home & landing page module designs/functionality.

**Parent Issue:** #427 

### Sub-Tasks
- [ ] Review user research findings and outline potential items that might need changes
- [ ] Review list with design team/PM to determine next steps

### Notes

*Some complementary notes if necessary:*
* > Here goes a quote from an email
* Here goes whatever useful information can exist…
* More notes, or technical nuances/direction

**Comment by lburack** [comment]
### TLDR:
no specific action items to take. We should be thoughtful in image selection (particularly when choosing lots of stock images), continue to monitor mobile to understand menu usage (since the menu wasn't clickable in this test), and continue to gather feedback from our audiences overtime.

### Top Findings:

- Participants felt the site was informative, appeared easy to use, visually appealing, with an overall tone of hopefulness. One participant said the homepage was tasteful, compact, efficient, well-organized.
- Overall, organization and layout of pages was clear and easy to understand.
- Call to action items were clear on each page
   - On the Partners in Cancer Research landing page though, there are 3 blue button links that are redundant of the cards on the page. This is a pattern used on other landing pages but b/c PiCR only has 3 subpages, the blue button links are more obviously redundant. On our other landing pages, we are using those blue button links to call out some of the most important content people may be looking for; that's not necessary with PiCR having so few subsections. 
   **- ACTION ITEM:** ensure blue button links are optional for landing pages. Provide guidance that they should be used to call out specific pages when there are multiple subsections.
- Several participants mentioned that they were happy to see diversity in the types of people shown in the images on the site, as well as the call out to Equity and Inclusion Program.

### Areas for Continued Exploration:
- Navigation was easier on desktop where participants could see the main navigation bar. When asked where they would go for the next task, mobile participants scrolled up and down the page. Some remarked that they would use the menu or search bar, but those items were not available for them to use. 
   - **ACTION ITEM:** We should continue to test the mobile experience as we design/build out more aspects of the site. Mobile issues for this test were likely due to the menu not being clickable, but we should monitor and continue to understand how the menu is used to navigate between pages on the site and if any usability issues exist/persist.
- Most participants responded well to the images, but some felt the photos were almost "glorifying" cancer in a way, with staged pictures of people looking happy. 
   - **ACTION ITEM:** We should be cognizant of the photos we select, particularly when a page is mostly stock images. But ultimately this is somewhat subjective and we heard most participants feel the site was hopeful.
- Most participants understood that the site was for several audiences. Some participants didn't understand why the site was structured this way and felt it was strange to have a researcher section on the same page as patients/caregivers. Upon landing on the Grants & Training landing page, one participant said it seemed like this landing page should be its own website — one site for cancer information and one for researchers.
   - **ACTION ITEM:** None for now. NCI stakeholders have articulated that those impacted by cancer and researchers are important audiences to NCI. We should continue to monitor how audiences respond to this and if it impacts how they perceive our website, particularly with patient/caregiver/interested public audiences who may think the site is not for them.
